--- 1/linux-tbs-drivers/v4l/Makefile	2012-03-31 18:40:13.277996318 +0100
+++ linux-tbs-drivers/v4l/Makefile	2012-03-31 18:40:32.137996318 +0100
@@ -179,7 +179,7 @@
   inst-m	:= $(obj-m)
 endif
 
-v4l_modules := $(shell /sbin/lsmod|cut -d' ' -f1 ) $(patsubst %.ko,%,$(inst-m))
+v4l_modules := $(shell /bin/lsmod|cut -d' ' -f1 ) $(patsubst %.ko,%,$(inst-m))
 
 #################################################
 # locales seem to cause trouble sometimes.

