--- linux-tbs-drivers.orig/v4l/Makefile	2018-10-05 18:17:20.009794551 +0100
+++ linux-tbs-drivers/v4l/Makefile	2018-10-05 18:17:39.889866457 +0100
@@ -201,13 +201,10 @@
 #################################################
 # installation invocation rules
 
-modules_install install:: media-install firmware_install
+modules_install install:: media-install
 
 remove rminstall:: media-rminstall
 
-firmware_install::
-	make -C firmware install
-
 #################################################
 # Compiling preparation rules
 
